src/components/Complete/index.js

```diff
@@ -10,11 +10,13 @@ class Complete extends Component {
     this.props.nextStep()
   }
 
-  render ({i18n}) {
+  render ({message, submessage, i18n}) {
+    const completeMessage = message ? message : i18n.t('complete.message')
+    const completeSubmessage = submessage ? submessage : i18n.t('complete.submessage')
```

Comment:
pretty minor, but these could just be `x = x || i18n.t('...')` instead, right? Save a bit of typing